src/testers.jl

```diff
@@ -97,9 +97,14 @@ function _make_jvp_call(fdm, f, xs, ẋs, ignores)
     ignores = collect(ignores)
     all(ignores) && return ntuple(_->nothing, length(xs))
     sigargs = zip(xs[.!ignores], ẋs[.!ignores])
-    return jvp(fdm, f2, sigargs...)
+    return to_composite(jvp(fdm, f2, sigargs...))
 end
 
+# remove after https://github.com/JuliaDiff/FiniteDifferences.jl/issues/97
```

Comment:
using `#TODO` causes it to get picked up by various automated tooling
```suggestion
#TODO remove after https://github.com/JuliaDiff/FiniteDifferences.jl/issues/97
```

Reply:
also maybe a comment about this fixing up FiniteDiff sometimes returning Tuples etc rathter than valid diff types as part of that TODO.
to explai what this function does?